manticore/core/state.py

```diff
@@ -76,10 +77,12 @@ def __init__(self, constraints, platform, **kwargs):
         self._input_symbols = list()
         self._child = None
         self._context = dict()
-        self._init_context()
         ##################################################################33
         # Events are lost in serialization and fork !!
         self.forward_events_from(platform)
+        
+        #FIXME(felipe) This should go into some event callback in a plugin (start_run?)
+        self._init_context()
```

Comment:
Should we update this to use new plugin events?

Reply:
Not sure we'll have the time to merge plugins. (I vote yes anyway!)